contents/LoadPriiloader.oscmeta

```diff
@@ -0,0 +1,15 @@
+{
+    "information": {
+        "name": "Load Priiloader",
+        "author": "DacoTaco",
+        "category": "utilities",
+        "peripherals": ["Wii Remote"],
+        "version": "auto"
+    },
+    "source": {
+        "type": "github_release",
+        "format": "zip",
+        "repository": "DacoTaco/priiloader",
+        "file": "LoadPriiloader.zip"

```

Comment:
github_release only pulls from latest stable github release, so this does not work.

Reply:
no u